src/components/jellyfinActions.ts

```diff
@@ -292,6 +306,16 @@ export function getPlaybackInfo(
     });
 }
 
+/**
+ * @param item
+ * @param playSessionId
+ * @param maxBitrate
+ * @param deviceProfile
+ * @param startPosition
+ * @param mediaSource
+ * @param audioStreamIndex
+ * @param subtitleStreamIndex
+ */
```

Comment:
maybe remove this too, then me or someone who actually documents the function can take away the warning